cmd/zoekt-webserver/main.go

```diff
@@ -481,8 +481,16 @@ func (s *loggedSearcher) log(ctx context.Context, q query.Q, opts *zoekt.SearchO
 			sglog.Duration("opts.MaxWallTime", opts.MaxWallTime),
 			sglog.Int("opts.MaxDocDisplayCount", opts.MaxDocDisplayCount),
 		)
+
 	if err != nil {
-		logger.Error("search failed", sglog.Error(err))
+		switch err {
+		case context.Canceled:
```

Comment:
`err` could in principle we wrapped. Shouldn't we check with `errors.Is(err, context.Canceled)` instead?

Reply:
Ah good point